default.json

```diff
@@ -18,7 +18,7 @@
         "safe_logging": false,
         "private_build": true,
         "development_api_enabled": true,
-        "mls_support_enabled": true,
+        "mls_support_enabled": false,
```

Comment:
This disables MLS completely we won't even look at the feature config.

Reply:
Yes, correct. Updated it correctly.